cnv_common_tasks.wdl

```diff
@@ -0,0 +1,598 @@
+task PreprocessIntervals {
+    File? intervals
+    File? blacklist_intervals
+    File ref_fasta
+    File ref_fasta_fai
+    File ref_fasta_dict
+    Int? padding
+    Int? bin_length
+    File? gatk4_jar_override
+
+    # Runtime parameters
+    String gatk_docker
+    Int? mem_gb
+    Int? disk_space_gb
+    Boolean use_ssd = false
+    Int? cpu
+    Int? preemptible_attempts
+
+    Int machine_mem_mb = select_first([mem_gb, 2]) * 1000
+    Int command_mem_mb = machine_mem_mb - 500
+
+    # Determine output filename
+    String filename = select_first([intervals, "wgs"])
+    String base_filename = basename(filename, ".interval_list")
```

Comment:
We _said_ we supported GATK-style interval lists, but if they have a `.interval_list` suffix they won't parse.  (So says Louis.) I can't think of an easy, elegant way to handle both in the WDL, so maybe we should just change the requirements in the readme?

Reply:
This is just to determine the filename of the output, which is always a Picard interval list with the `.interval_list` suffix.  The input `intervals` can be in any format supported by the GATK engine.